seed/challenges/02-javascript-algorithms-and-data-structures/basic-javascript.json

```diff
@@ -2636,7 +2636,7 @@
         "<blockquote>function plusThree(num) {<br>  return num + 3;<br>}<br>var answer = plusThree(5); // 8</blockquote>",
         "<code>plusThree</code> takes an <dfn>argument</dfn> for <code>num</code> and returns a value equal to <code>num + 3</code>.",
         "<hr>",
-        "Create a function <code>timesFive</code> that accepts one argument, multiplies it by <code>5</code>, and returns the new value."
+        "Create a function <code>timesFive</code> that accepts one argument, multiplies it by <code>5</code>, and returns the new value. You can call the <code>timesFive</code> function to test it."
```

Comment:
The seed code has the seeded function `minusSeven` so the comment wouldn't be accurate, as you would be testing the given function. ~~Maybe modify the text description to be~~

> ~~You can call the `timesFive` function to test it, similar to how the `minusSeven` function is called in the editor.~~

On second thought, maybe we should just remove this text because as @Greenheart has mentioned, the `console.log` of the `timesFive` function won't appear in the on-site console, although it can be seen in the browser's console.